src/fileTransformers/engineType.ts

```diff
@@ -14,11 +14,8 @@ export const engineType: FileTransformer = (params) => {
     }
   }
   if (file.path === 'package.json') {
-    if (parameters.engineType) {
-      content = content.replace(
-        `"@prisma/client": "3.1.1"`,
-        `"@prisma/client": "3.1.0-integration-data-proxy-engine.1"`
-      )
+    if (parameters.engineType && parameters.engineType === 'dataproxy') {
+      content = content.replace(`"@prisma/client": "3.1.1"`, `"@prisma/client": "dataproxy"`)
```

Comment:
Is there an API from GH or npm that allows you to fetch the most recent release? It would be better than hard coding the `@prisma/client` version. 🤔

Just curious. 

https://docs.github.com/en/rest/reference/repos#list-releases

Reply:
That's the idea behind NPM tags. An NPM tag is a pointer to a version (any version).

By default, all releases of a package on NPM have the `latest` tag, which automatically always points to the last published semver version. But we can create our own (which is what `dataproxy` is). When we do, we have to make sure that every time we release something new, we point our tag to that version.

So when you set the version of `@prisma/client` to `latest` in your package.json, you're saying that I want whatever version this tag points to at this moment. This is why if _we_ keep the `dataproxy` tag pointing to whatever version we want, we will never have to change this entry in our package.json (or in this repo)